### PR TITLE
Issue #193

### PR DESCRIPTION
**Pull request by sonAndrew**
In this change I added 3 new photos to the images folder. They are **"OLE_Edu.jpg"**, **"OLE_International"** this is used as hero image, and **"OLE_RPi.png"** this is used in the body when talking about the **raspberypi**. The next changes are in the body where I **"highlighted"** important information making reading each section easier. I also added **qouted sections** for asides to give the reader extra help like pointing to the **"FAQ"**, the **"leaders"**, and **"OLEs moto"**. Finally, I changed the **"click here"** link to **"Get Started"**.

### CODE REVIEW

**Comment by pattanawadee88** [comment]
Could you please put the rawgit link for us?

File: index.md [hunk 1]
```diff
@@ -1,12 +1,45 @@
-# Welcome!
+# Welcome & Congratulations!
+### You just took your first step towards becoming a Virtual Intern!
 
-This MDwiki serves as an information repository for treehouses, a project of the Open Learning Exchange.  Please feel free to join us on our [Gitter chat](https://gitter.im/treehouses/Lobby).
+<br />
```
Comment (dogi) [inline index.md:4]: @sonAndrew please do not use HTML

File: index.md [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
 
-The treehouses project is divided into several parts to meet the needs of different users, including those participating in our Virtual Intern program.
+![OLE International Team](./images/OLE_International_Team.jpg)
+###### The OLE International Team. [Left to Right] Rabi, April Stone, Stefan Unterhauser, Rory Phimister, Denisia Mangue, Richard R. Rowe, Boyah J. Farah and Leonard Maximus Ireneous Mensah not pictured.
 
-If you are a new Virtual Intern, please go to the "Virtual Interns" tab and go to "Introduction / First Steps", or [click here](pages/vi/firststeps.md) to get started!  Please remember that many team members are available to help you on Gitter chat.
+<br />
```
Comment (dogi) [inline index.md:9]: this is not up to date ... lets just remove it ;)

File: index.md (hunk 1 above)
Comment (sonAndrew) [reply index.md:4]: Why not? I only used it for spacing and one image.

File: index.md (hunk 2 above)
Comment (sonAndrew) [reply index.md:9]: What do you mean? I believe I changed that portion also.

**Comment by CalebProvost** [comment]
@sonAndrew 

As @pattanawadee88 has mentioned, don't forget to post a "raw.githack" link so we can easily see the rendered differences to help us review your code.

Also, to help you understand the reason why we don't use HTML: consistency.
Practically everything we use for our publishing will be MarkDown. Only when its absolutely necessary (MD can't render what's needed and workarounds are not doable) do we make an exception for HTML and even then, @dogi has to be convinced of it.
This keeps things consistent and uniform.
Your use of  breaklines (`<br />`) are done with markdown as two "new lines" so the use of html to do it isn't needed.

https://raw.githack.com/sonAndrew/sonAndrew.github.io/master/#!index.md

**Review by pattanawadee88 — CHANGES_REQUESTED** [review]
Also, you should not work on your master branch. You have to create a new branch from your master branch and work on that branch.

**Pull request closed without merging**